src/openapi/streaming/streaming.ts

```diff
@@ -473,10 +485,18 @@ class Streaming extends MicroEmitter<EmittedEvents> {
 
                 for (let i = 0; i < this.subscriptions.length; i++) {
                     this.subscriptions[i].onConnectionAvailable();
+                    if (this.shouldSubscribeBeforeStreamingSetup) {
+                        // trigger onActivity of subscribed subscriptions before streaming setup to avoid reset by orphan finder
+                        this.subscriptions[i].onHeartbeat();

```

Comment:
When we call `onHeartbeat` and a subscription has `STATE_SUBSCRIBE_REQUESTED` state, we log `Received heartbeat for a subscription we havent subscribed to yet`. Is it possible that a subscription is in that state when we cal `onHeartbeat` here? If yes, then we will get some falsy log messages in that case.

Reply:
yes it can 
i have passed a flag to not to log the message when calling onHeartbeat for the subscribe for streaming case